### PR TITLE
Tslint rules implementation

### LINKED ISSUE #157: Enhance TsLint rules

### PR DESCRIPTION
**Pull request by corentingiraud**
### Description of your PR

I've followed the back rules (ESlint). I've implemented some rules described in [this medium post](https://medium.com/burak-tasci/angular-tslint-rules-a-configuration-preset-for-both-tslint-codelyzer-8b5fa1455908). Will close #157 

### Pre-review TODO

- [x] PR is rebased on top of `origin/master` 
- [ ] PR provide new tests for new behaviors
- [x] PR doesn't introduce commented out code
- [x] PR has been tested locally on the main use cases
- [ ] Documentation have been updated if necessary

### CODE REVIEW

File: client/e2e/app.e2e-spec.ts [hunk 1]
```diff
@@ -1,14 +1,14 @@
 import { AppPage } from './app.po';
```
Comment (Embraser01) [inline client/e2e/app.e2e-spec.ts]: Let's delete this file, it doesn't pass test anyway 😃

File: client/src/app/menu/menu.component.ts [hunk 2]
```diff
@@ -30,72 +29,72 @@ export class MenuComponent implements OnDestroy, OnInit {
...
     ];
 
     mobileQuery: MediaQueryList;
     sidenavQuery: MediaQueryList;
     user: ConnectedUser;
 
-    private _mobileQueryListener: () => void;
-    private _sidebavQueryListener: () => void;
+    _mobileQueryListener: () => void;
+    _sidebavQueryListener: () => void;
```
Comment (Embraser01) [inline client/src/app/menu/menu.component.ts]: Why did you remove the private keyword ?

File: client/src/app/services/open-services/open-services.component.ts [hunk 3]
```diff
@@ -1,66 +1,67 @@
...
+            this.servicesFormArray,
         ]);
         this.generalFormGroup = this.formBuilder.group({
-            generalFormArray: this.generalFormArray
+            generalFormArray: this.generalFormArray,
         });
     }
 
-    addServiceForm(categoryId: Number, nbMax: Number, startAt: Moment, endAt: Moment) {
+    addServiceForm(nbMax: Number, startAt: Moment, endAt: Moment): void { // TODO: categoryId: Number
```
Comment (Embraser01) [inline client/src/app/services/open-services/open-services.component.ts]: TODO ? `categoryId` does not exist anymore, we can delete it no ?

Comment (kistora) [reply client/src/app/services/open-services/open-services.component.ts]: +1

File: client/src/app/menu/menu.component.ts (hunk 2 above)
Comment (kistora) [reply client/src/app/menu/menu.component.ts]: False positive TSLint error...

File: client/e2e/app.e2e-spec.ts (hunk 1 above)
Comment (kistora) [reply client/e2e/app.e2e-spec.ts]: Right
